docs/search_config.md

```diff
@@ -0,0 +1,19 @@
+# Search Algorithm Configuration
+
+The calibration process utilizes a genetic algorithm. The configuration file includes parameters that allow users to tune the genetic algorithm for optimal performance.
+
+- **population_size**: Defines the number of candidate solutions in each generation. A larger population increases diversity and enhances the search, but require more computational resources. Very small populations may lead to premature convergence.
+- **generations**: Specifies the maximum number of iterations the algorithm can perform. Increasing this value allows more opportunities for improvement, but leads to longer runtimes. The calibration process will terminate early if an optimal solution is found before reaching this limit.
```

Comment:
I think we should not use the word `optimal`, and instead say something like: "if a solution that meets the user's criteria is found..."

Reply:
"Very small populations may lead to premature convergence." I don't understand this phrasing. Isn't premature convergence a good thing? You want it to converge as quickly as possible. I am probably misunderstanding something here.